api/src/wfl/service/firecloud.clj

```diff
@@ -20,6 +20,12 @@
       (http/get {:headers (auth/get-auth-header)})
       util/response-body-json))
 
+(defn get-workspace
+  "Returns the given workspaceNamespace/workspaceName`"
+  [workspace]
+  {:pre [(some? workspace)]}
+  (get-workspace-json workspace))
```

Comment:
Maybe string? instead of some? or just inline `(get-workspace-json workspace)` where necessary.

Reply:
Changed to string.